addons/purchase/models/purchase.py

```diff
@@ -352,6 +352,18 @@ def button_unlock(self):
     @api.multi
     def button_done(self):
         self.write({'state': 'done'})
+    
+    @api.multi
```

Comment:
don't need `@api.multi`

Reply:
Thank you @fmdl 
the issue has been fixed https://github.com/odoo/odoo/pull/55428/commits/2e4ddc45ec51eae8dcd63ca6627d66ea942c79d8

Please review